freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/user/UserV1Endpoint.java

```diff
@@ -77,4 +79,18 @@ SyncOperationStatus getSyncOperationStatusInternal(
     @ApiOperation(value = UserOperationDescriptions.ENVIRONMENT_USERSYNC_STATE, notes = UserNotes.USER_NOTES, produces = MediaType.APPLICATION_JSON,
             nickname = "getEnvironmentUserSyncStateV1")
     EnvironmentUserSyncState getUserSyncState(@QueryParam("environmentCrn") @NotEmpty String environmentCrn);
+
+    @POST
+    @Path("presync")
```

Comment:
should these be "internal/presync" and "internal/postsync" instead? I know we've added things to the "internal" path when the endpoint duplicates an existing non-internal endpoint, but I'm not sure about for other endpoints.

Reply:
this could be used as non internal only, by removing the internal only annotation and adding checkpermission, with the right permission